Mathlib/Analysis/Calculus/Inverse.lean

```diff
@@ -456,12 +455,31 @@ def toLocalHomeomorph (hf : ApproximatesLinearOn f (f' : E →L[𝕜] F) s c)
   continuous_invFun := hf.inverse_continuousOn hc
 #align approximates_linear_on.to_local_homeomorph ApproximatesLinearOn.toLocalHomeomorph
 
+@[simp]
```

Comment:
just to make sure all is good - they were moved up because of `toHomeomorph`, and otherwise unchanged, right?